java-bigquerydatatransfer/.OwlBot-hermetic.yaml

```diff
@@ -20,52 +20,7 @@ deep-remove-regex:
 - "/java-bigquerydatatransfer/samples/snippets/generated"
 
 deep-preserve-regex:
-- "/java-bigquerydatatransfer/google-.*/src/test/java/com/google/cloud/.*/v.*/it/IT.*Test.java"
```

Comment:
Do we want to delete this IT?

Reply:
There is actually no integration tests for bigquerydatatransfer, so this is like a not-directly-related clean up. I can revert this as well if it is causing confusion.

Reply:
Ah ok. If there are no ITs to being with then it doesn't matter. I'm fine with removing